tests/testthat/test-notes.R

```diff
@@ -52,6 +53,19 @@ test_that("showing notes", {
     add_recipe(clean_rec)
 
   res_clean <- clean_wflow %>% fit_resamples(rs)
-  expect_snapshot(show_notes(.Last.tune.result
-                             ))
+  expect_snapshot(show_notes(.Last.tune.result))
+
+  # Get cli lines right
+  set.seed(1)
+  dat <- modeldata::sim_classification(150, intercept = 15)
+  rs <- rsample::vfold_cv(dat)
+  fit_lr <-
+    parsnip::logistic_reg() %>%
+    fit_resamples(
+      class ~ .,
+      rs,
+      metrics = yardstick::metric_set(accuracy, kap, recall, precision)
```

Comment:
Why not leave `metrics = NULL`? This way you would get a more minimal test, and you get a slightly larger output

Reply:
That was mostly there to help me trigger the warnings.